plugins/net_plugin/net_plugin.cpp

```diff
@@ -3820,7 +3820,9 @@ namespace eosio {
            my->acceptor->bind(listen_endpoint);
            my->acceptor->listen();
          } catch (const std::exception& e) {
-           elog( "net_plugin::plugin_startup failed to bind to port ${port}", ("port", listen_endpoint.port()) );
+           fc_elog(logger, "net_plugin::plugin_startup failed to bind to port ${port}." 
+		 "Please check if the port is likely in use by another application." 
+		 "Either choose a different port or stop that application", ("port", listen_endpoint.port()) );
```

Comment:
I think you could use the exact error message recommended by Kevin, i.e., ".. . The port is likely in use by another application."

Reply:
done, see another commit